src/storage/mod.rs

```diff
@@ -792,7 +820,29 @@ impl<E: Engine, L: LockManager> Storage<E, L> {
                         .get(CMD)
                         .observe(command_duration.saturating_elapsed_secs());
```

Comment:
We have so many duplicated `elapsed` counts and `instant::now` calls, and they actually serve similar purposes. I'm looking forward to seeing refinements later. For example, some underlying time measurements provides both metrics and per-request statistics.

Reply:
I listed it as a separate task in the roadmap issue.